### DISCUSSION TITLE
Will there be a Time Picker?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [adobe/react-spectrum#2773] [Q&A] [answered]
I just saw the latest storybook [here](https://reactspectrum.blob.core.windows.net/reactspectrum/0a6bb97e6d63c6d8d23845ca5ec2f29076e4fa0b/storybook-17/index.html?path=/story/date-and-time-timefield--hourcycle-12) & I only see a **Time Field**.

Will there be a **Time Picker**?

I like the one used in [react-datetime](https://www.npmjs.com/package/react-datetime). See [demo](https://codesandbox.io/s/boring-dew-uzln3)

although, i do think **Time Field** feels much simpler but still asking.

Great work, love it :)

**Comment by devongovett** [comment, marked as answer]
I don't think we currently have a plan to build a more heavy UI for time picking (e.g. that example, or material's fancy analog clock). In my opinion, the time field is simple and works well enough so I don't think more is necessary. Nothing stopping someone from using our libraries to build that though!
